source/install/install-mmte-helm-gitlab-helm.rst

```diff
@@ -0,0 +1,233 @@
+..  _install-mmte-helm-gitlab-helm:
+
+How to deploy Mattermost Team Edition Helm Chart in the GitLab Helm Chart deployment
+=====================================================================================
+
+This document will describe how to deploy Mattermost Team Edition Helm Chart in an existing GitLab Helm Chart deployment.
+
+You will need:
+  - A running Kubernetes cluster
+  - A running GitLab Helm Chart
+  - The name of the secret that holds the Postgres password. If you installed the Gilab Helm  Chart using the chart name as ``gitlab``, then the secret name is ``gitlab-postgresql-password``
+  - The name of the secret that holds the Minio keys. If you installed the Gilab Helm Chart using the chart name as ``gitlab``, then the secret name is ``gitlab-minio-secret``
+  - The service name for Postgres and the port. If you installed the Gilab Helm Chart using the chart name as ``gitlab`` in the ``default`` namespace, then the service name is ``gitlab-postgresql`` and port is ``5432``
+  - The service name for Minio and the port. If you installed the Gilab Helm Chart using the chart name as ``gitlab`` in the ``default`` namespace, then the service name is ``gitlab-minio-svc`` and port is ``9000``
+  - The names of ``kubernetes.io/ingress.class``, ``kubernetes.io/ingress.provider`` and ``certmanager.k8s.io/issuer``
+
+**Deploy GitLab Helm Chart**
+----------------------------
+
+To deploy Gitlab Helm Chart, you can follow the instructions described `here <https://docs.gitlab.com/ee/install/kubernetes/gitlab_chart.html>`_.
+
+A light way to install is:
+
+.. code-block:: bash
+
+  $ helm upgrade --install gitlab gitlab/gitlab \
+    --timeout 600 \
+    --set global.hosts.domain=<YOUR.DOMAIN> \
+    --set global.hosts.externalIP=<EXTERNAL.IP> \
+    --set certmanager-issuer.email=<EMAIL>
+
+- **<YOUR.DOMAIN>**: your desired domain, eg. gitlab.example.com
+- **<EXTERNAL.IP>**: the external ip pointing to your k8s cluster
+- **<EMAIL>**: email to register in the Let's Encrypt to get the TLS certificates
+
+After all pods get in the running state, you can log in to GitLab.
+
+**Create an OAuth application with GitLab**
+--------------------------------
+
+To create the OAuth app to allow Mattermost to use GitLab as the authentication provider, please follow the instructions `here <https://docs.mattermost.com/administration/config-settings.html#gitlab>`_.
+
+Please take note of the ``Application ID``, ``Application Secret Key``, ``User API Endpoint``, ``Auth Endpoint`` and ``Token Endpoint``, as these values will be used later when deploying Mattermost.
+
+**Deploy Mattermost Team Edition Helm Chart**
+---------------------------------------------
+
+Requirements:
+  - Mattermost Team Edition Helm Chart Version => 1.4.0
+
+To deploy Mattermost together with GitLab Helm Chart, you need to set some values in the ``values.yaml`` for the InitContainer and Environment Variables. You also need to disable the ``MySql`` chart.
```

Comment:
Is it `MySql` chart or `Postgres` chart?

Reply:
MySql